owl/CROMA.owl

```diff
@@ -0,0 +1,438 @@
+                <owl:someValuesFrom rdf:resource="http://www.ease-crc.org/ont/CROMA.owl#Action_Designator_Design"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:label>Action Designator</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/CROMA.owl#ActionDesignatorDescription -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/CROMA.owl#ActionDesignatorDescription">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/CROMA.owl#ActionDesignator"/>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/CROMA.owl#ActionDesignatorPerformed -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/CROMA.owl#ActionDesignatorPerformed">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/CROMA.owl#ActionDesignator"/>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/CROMA.owl#ActionDesignatorResolved -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/CROMA.owl#ActionDesignatorResolved">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/CROMA.owl#ActionDesignator"/>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/CROMA.owl#Action_Designator_Design -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/CROMA.owl#Action_Designator_Design">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/CROMA.owl#Designator_Design"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#describes"/>
+                <owl:someValuesFrom rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Action"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:label xml:lang="en">Action Designator Design</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/CROMA.owl#Board -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/CROMA.owl#Board">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#DesignedTool"/>
+        <rdfs:comment>A board is a flat piece of wood or other hard material. (PhysicalObject)</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/CROMA.owl#CRAM_Cognitive_Architecture -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/CROMA.owl#CRAM_Cognitive_Architecture">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Software_Configuration"/>
+        <rdfs:label xml:lang="en">CRAM Cognitive Architecture</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/CROMA.owl#CRAM_Designator -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/CROMA.owl#CRAM_Designator">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/CROMA.owl#Designator"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
```

Comment:
This restriction should say that a CRAM designator is resolved_to only a CRAM designator. Not all designators are resolved, but those that are can only be resolved to designators.